File: app/src/main/java/com/zuk0/gaijinsmash/riderz/data/local/room/dao/EtdDao.java
package com.zuk0.gaijinsmash.riderz.data.local.room.dao;

import androidx.room.Dao;
import androidx.room.Insert;
import androidx.room.Query;

import com.zuk0.gaijinsmash.riderz.data.local.entity.etd_response.EtdXmlResponse;

import static androidx.room.OnConflictStrategy.REPLACE;

@Dao
public interface EtdDao {
    @Insert(onConflict = REPLACE)
    void save(EtdXmlResponse etd);

    //@Query("SELECT * from advisories where id = :id")
    //EtdXmlResponse getEtdById(int id);
}

File: app/src/main/java/com/zuk0/gaijinsmash/riderz/data/local/room/dao/EtdDao.kt
package com.zuk0.gaijinsmash.riderz.data.local.room.dao

import androidx.room.Dao
import androidx.room.Insert
import androidx.room.OnConflictStrategy
import com.zuk0.gaijinsmash.riderz.data.local.entity.etd_response.EtdXmlResponse

@Dao
interface EtdDao {
    @Insert(onConflict = OnConflictStrategy.REPLACE)
    fun save(etd: EtdXmlResponse?) //@Query("SELECT * from advisories where id = :id")
//EtdXmlResponse getEtdById(int id);
}